ui/apps/platform/src/Containers/VulnMgmt/Reports/Components/ScopeName.tsx

```diff
@@ -1,64 +1,63 @@
 import React, { ReactElement } from 'react';
-import { Button, ButtonVariant, Spinner } from '@patternfly/react-core';
+import { useHistory } from 'react-router-dom';
+import { Alert, Button, ButtonVariant } from '@patternfly/react-core';
 
 import LinkShim from 'Components/PatternFly/LinkShim';
 import { getEntityPath } from 'Containers/AccessControl/accessControlPaths';
 import useFeatureFlags from 'hooks/useFeatureFlags';
-import useFetchScopes from 'hooks/useFetchScopes';
-import useCollection from 'Containers/Collections/hooks/useCollection';
-import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
 import { collectionsBasePath } from 'routePaths';
+import { ReportScope } from 'hooks/useFetchReport';
 
 type ScopeNameProps = {
-    scopeId: string;
+    reportScope: ReportScope | null;
 };
 
-function AccessScopeName({ scopeId }: ScopeNameProps): ReactElement {
-    const scopesResult = useFetchScopes();
-
-    if (!scopeId) {
+function AccessScopeName({ reportScope }: ScopeNameProps): ReactElement {
+    if (!reportScope) {
         return <em>No resource scope specified</em>;
     }
 
-    const fullScope = scopesResult.scopes.find((scope) => scope.id === scopeId);
-
-    if (scopesResult.isLoading) {
-        return <Spinner isSVG size="md" />;
-    }
-
-    if (scopesResult.error) {
-        return <span>Error getting scope info. {getAxiosErrorMessage(scopesResult.error)}</span>;
-    }
-
-    const url = getEntityPath('ACCESS_SCOPE', scopeId);
+    const url = getEntityPath('ACCESS_SCOPE', reportScope.id);
 
     return (
         <Button variant={ButtonVariant.link} isInline component={LinkShim} href={url}>
-            {fullScope?.name || scopeId}
+            {reportScope.name}
         </Button>
     );
 }
 
-function CollectionScopeName({ scopeId }: ScopeNameProps): ReactElement {
-    const { data, loading, error } = useCollection(scopeId);
+function CollectionScopeName({ reportScope }: ScopeNameProps): ReactElement {
+    const history = useHistory();
 
-    if (!scopeId) {
+    if (!reportScope) {
         return <em>No report scope specified</em>;
     }
 
-    if (loading) {
-        return <Spinner isSVG size="md" />;
-    }
-
-    if (error) {
-        return <span>Error getting scope info. {getAxiosErrorMessage(error)}</span>;
+    if (reportScope.type === 'AccessControlScope') {
+        return (
+            <Alert
+                isInline
+                variant="danger"
+                title="The report scope for this configuration could not be migrated to a collection"
+                actionLinks={
```

Comment:
Does action link depend on READ_WRITE_ACCESS for WorkflowAdministration resource?

This component is for detail view which requires only READ_ACCESS before this change, correct?

Reply:
Fixed, added a screenshot as well.